<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Dificult</string>
   </property>
   <widget class="QRadioButton" name="D_Easy">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="D_norm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="D_hard">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Speed</string>
   </property>
   <widget class="QRadioButton" name="S_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 per second</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="S_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3 per second</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="S_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5 per second</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Field size</string>
   </property>
   <widget class="QSpinBox" name="W_size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>200</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="H_size">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>200</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Widght</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Game Name</string>
   </property>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Accept">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
